pkg/cli/api/version.go

```diff
@@ -14,45 +14,42 @@ type VersionInfo struct {
 
 // Version include the version and commit.
 type Version struct {
-	Version      string `json:"version" yaml:"version"`
-	GitCommit    string `json:"gitCommit" yaml:"gitCommit"`
-	IsDevVersion bool   `json:"isDevVersion" yaml:"isDevVersion"`
+	Version   string `json:"version" yaml:"version"`
+	GitCommit string `json:"gitCommit" yaml:"gitCommit"`
 }
 
 // GetVersion get client, server and openapi version.
-func GetVersion(sc *config.Config) (*VersionInfo, error) {
+func GetVersion(sc *config.Config) *VersionInfo {
 	// Client version.
 	info := &VersionInfo{
 		ClientVersion: Version{
-			Version:      version.Version,
-			GitCommit:    version.GitCommit,
-			IsDevVersion: version.IsDevVersion(),
+			Version:   version.Version,
+			GitCommit: version.GitCommit,
 		},
 	}
 
 	if sc != nil && sc.Reachable {
 		// Server version.
 		sv, err := sc.ServerVersion()
 		if err != nil {
-			return nil, err
+			// Return client version if server version is not reachable.
+			return info
```

Comment:
Is there any message showing the server is not reachable?

Reply:
Prefer just show the client version while the server isn't reachable. While the user run other commands need interact with the server would show the unreachable error.